applications/nrf5340_audio/src/audio_module/generic_module/aobj_api.h

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright(c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef _AOBJ_API_H_
+#define _AOBJ_API_H_
+
+#include <zephyr/kernel.h>
+#include <stddef.h>
+#include <stdbool.h>
+#include <stdint.h>
+
+/**
+ * @brief Value to define the data type carried by the object.
+ *
+ */
+enum aobj_type {
+	/*! The audio data object is raw PCM */
+	AOBJ_TYPE_PCM = 0,
+
+	/*! The audio data object is coded data */
+	AOBJ_TYPE_CODED = 1,
+};
+
+/**
+ * @brief Value to define the PCM type carried by the object.
+ *
+ */
+enum aobj_pcm_type {
+	/*! The audio data object contains samples in linear format */
+	AOBJ_PCM_TYPE_LINEAR = 0,
+
+	/*! The audio data object contains samples in ulaw format */
+	AOBJ_PCM_TYPE_ULAW = 1,
+
+	/*! The audio data object contains samples in Alaw format */
+	AOBJ_PCM_TYPE_ALAW = 2
+};
+
+/**
+ * @brief Value to define the coding type carried by the object.
+ *
+ */
+enum aobj_coding_type {
+	/*! The audio data object is coded in LC3 */
+	AOBJ_CODING_TYPE_LC3 = 0,
```

Comment:
Suggest that we have 0 = Invalid/not set to easily detect any issues. We can also consider using the definitions from BAP where LC3 = 6.

Reply:
We can as the code is now very closely linked to LE audio.